docs/data-sources/service.md

```diff
@@ -0,0 +1,204 @@
+
+Encore provisioned service information
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `name` (String) The name of the Encore resource
+
+### Optional
+
+- `env` (String) The environment of the Encore resource. Defaults to the provider environment
+
+### Read-Only
+
+- `aws_fargate_task_definition` (Attributes) (see [below for nested schema](#nestedatt--aws_fargate_task_definition))
+- `cluster_ip` (Attributes) (see [below for nested schema](#nestedatt--cluster_ip))
+- `gcp_cloud_run` (Attributes) (see [below for nested schema](#nestedatt--gcp_cloud_run))
+- `k8s_container` (Attributes) (see [below for nested schema](#nestedatt--k8s_container))
+
+<a id="nestedatt--aws_fargate_task_definition"></a>
+### Nested Schema for `aws_fargate_task_definition`
+
+Read-Only:
+
+- `arn` (String)
+- `execution_role` (Attributes) (see [below for nested schema](#nestedatt--aws_fargate_task_definition--execution_role))
+- `service` (Attributes) (see [below for nested schema](#nestedatt--aws_fargate_task_definition--service))
+- `task_role` (Attributes) (see [below for nested schema](#nestedatt--aws_fargate_task_definition--task_role))
+
+<a id="nestedatt--aws_fargate_task_definition--execution_role"></a>
+### Nested Schema for `aws_fargate_task_definition.execution_role`
+
+Read-Only:
+
+- `arn` (String)
+
+
+<a id="nestedatt--aws_fargate_task_definition--service"></a>
+### Nested Schema for `aws_fargate_task_definition.service`
+
+Read-Only:
+
+- `arn` (String)
+- `cluster` (Attributes) (see [below for nested schema](#nestedatt--aws_fargate_task_definition--service--cluster))
+- `security_groups` (List of Object) (see [below for nested schema](#nestedatt--aws_fargate_task_definition--service--security_groups))
+- `subnets` (List of Object) (see [below for nested schema](#nestedatt--aws_fargate_task_definition--service--subnets))
+
+<a id="nestedatt--aws_fargate_task_definition--service--cluster"></a>
+### Nested Schema for `aws_fargate_task_definition.service.cluster`
+
+Read-Only:
+
+- `arn` (String)
+
+
+<a id="nestedatt--aws_fargate_task_definition--service--security_groups"></a>
+### Nested Schema for `aws_fargate_task_definition.service.security_groups`
+
+Read-Only:
+
+- `id` (String)
+
+
+<a id="nestedatt--aws_fargate_task_definition--service--subnets"></a>
+### Nested Schema for `aws_fargate_task_definition.service.subnets`
+
+Read-Only:
+
+- `arn` (String)
+- `az` (String)
+- `vpc` (Object) (see [below for nested schema](#nestedobjatt--aws_fargate_task_definition--service--subnets--vpc))
+
+<a id="nestedobjatt--aws_fargate_task_definition--service--subnets--vpc"></a>
+### Nested Schema for `aws_fargate_task_definition.service.subnets.vpc`
+
+Read-Only:
+
+- `id` (String)
+
+
+
+
+<a id="nestedatt--aws_fargate_task_definition--task_role"></a>
+### Nested Schema for `aws_fargate_task_definition.task_role`
+
+Read-Only:
+
+- `arn` (String)
+
+
+
+<a id="nestedatt--cluster_ip"></a>
+### Nested Schema for `cluster_ip`
```

Comment:
Not sure what a `cluster_ip` is in this context

Reply:
it's the k8s cluster ip. Guess we can prefix it with k8s